tests/model_selection/test_incremental.py

```diff
@@ -188,22 +194,41 @@ def additional_calls(scores):
 
 
 @gen_cluster(client=True)
-def test_search(c, s, a, b):
+def test_search_basic(c, s, a, b):
     X, y = make_classification(n_samples=1000, n_features=5, chunks=(100, 5))
     model = SGDClassifier(tol=1e-3, loss="log", penalty="elasticnet")
 
     params = {"alpha": np.logspace(-2, 2, 100), "l1_ratio": np.linspace(0.01, 1, 200)}
 
-    search = IncrementalSearch(model, params, n_initial_parameters=10, max_iter=10)
+    search = IncrementalSearchCV(model, params, n_initial_parameters=20, max_iter=10)
     yield search.fit(X, y, classes=[0, 1])
 
-    assert search.history_results_
-    for d in search.history_results_:
+    assert search.history_
+    for d in search.history_:
         assert d["partial_fit_calls"] <= search.max_iter + 1
     assert isinstance(search.best_estimator_, SGDClassifier)
     assert search.best_score_ > 0
     assert "visualize" not in search.__dict__
     assert search.best_params_
+    assert search.cv_results_ and isinstance(search.cv_results_, dict)
+    assert {
+        "mean_partial_fit_time",
+        "mean_score_time",
+        "std_partial_fit_time",
+        "std_score_time",
+        "test_score",
+        "rank_test_score",
+        "model_id",
+        "params",
+        "partial_fit_calls",
+        "param_alpha",
+        "param_l1_ratio",
+    }.issubset(set(search.cv_results_.keys()))
+    assert all(isinstance(v, np.ndarray) for v in search.cv_results_.values())
+    assert search.cv_results_["test_score"][search.best_index_] >= np.percentile(
```

Comment:
I'd like to avoid this if possible. This seems like a recipe for randomly failing tests in the future.

Is there any way to reconstruct the best model from the output of `search.history_`?

Reply:
Yes. If the search is passive (not adaptive), `search.best_index_` is guaranteed to point to the highest scoring model (because there's only one dimension to the scores as mentioned in https://github.com/dask/dask-ml/pull/370#discussion_r224456867). I've implemented this change in 0374b02ffdfa1084115df17bb6303694b0a62b24.

To make the search passive, I set the default `decay_rate=0`. With the default `patience=False`, this means `IncrementalSearchCV` is completely passive by default (added to https://github.com/dask/dask-ml/pull/370#issue-216493783). 

This is the most basic thing and the first thing to try, so I think this should be the default.

Reply:
Given your previous work showing that adaptivity is almost always a good idea I recommend that we leave the default as-is.  If we should discuss this then lets open up a separate issue and eventually open up a separate PR.  It would be good not to increase the scope of this PR further if possible.